core/common/test/format/LocalDateTimeFormatTest.kt

```diff
@@ -224,6 +224,41 @@ class LocalDateTimeFormatTest {
         test(dateTimes, LocalDateTime.Formats.ISO)
     }
 
+    @Test
+    fun testBasicIso() {
+        val dateTimes = buildMap<LocalDateTime, Pair<String, Set<String>>> {
+            put(LocalDateTime(2008, 7, 5, 0, 0, 0, 0), ("20080705T000000" to setOf("20080705t000000")))
```

Comment:
Please also test the scenarios of parsing a string which does not include the seconds.